src/api/variations.ts

```diff
@@ -1,5 +1,19 @@
+import { IsString, IsNotEmpty, IsOptional } from 'class-validator'
 import apiClient from './apiClient'
-import { Variation } from './schemas'
+import { Feature, Variation } from './schemas'
+
+export class CreateVariationParams {
```

Comment:
I believe this class is no longer required given some work @emmawillis has been doing - zodios is supposed to auto-pickup all these validation decorators. But might need to check with her, or possibly she can update this on her ticket.